### ISSUE TITLE
[DAPI] - Handle Fitbit TZ

### ISSUE THREAD

**Issue by Goncharo** [metriport/metriport#678]
### Problem

From a cx:

---
Looking at the [Fitbit docs](https://dev.fitbit.com/build/reference/web-api/body/get-weight-log/#Response): their weight-log response includes separate date and time fields, but I don't see any detail about whether the time value is UTC, or local time.

https://community.fitbit.com/t5/Web-API-Development/Clarification-on-time-zones-and-date-time-values/td-p/925255#:~:text=There%20is%20no%20concept%20of,after%20the%20measurement%20was%20logged.

yeah, seems like there's a lot of frustration with how Fitbit handles timezones. Kinda ridiculous that they haven't improved anything in the 8 years since this post was written
---

### Suggested solution

See this: https://dev.fitbit.com/build/reference/web-api/developer-guide/application-design/#Time-Zones

TL;DR is the quickest thing to do would be to get the user's TZ from their profile on Fitbit endpoints that don't support the `timezone` query param, and convert to UTC.

Also, if we're not already, we should explicitly add the `timezone=UTC` query param on endpoints that support it.
